EpikCord/managers/channel_manager.py

```diff
@@ -12,7 +12,7 @@
     GuildStageChannel,
 )
 
-AnyChannel = Union[
+AnyChannel:TypeAlias = Union[
```

Comment:
`TypeAlias` has been introdced in python 3.10, which means you are discarding 2 python version (3.8 & 3.9) support when 3.8 eol is planed for october 2024